docs/repository-management.md

```diff
@@ -0,0 +1,134 @@
+
+When reviewing the issue backlog, here are some steps you can perform:
+
+- If it’s a bug report, test to confirm the report.
+- If the issue is missing labels, add some to better categorize it.
+- If the issue is duplicate of another already in the backlog, close the issue by commenting with “Duplicate of #<original-id>”. Add any relevant new details to the existing issue.
+- If there was conversation on the issue but no actionable steps identified, follow up with the participants to see what’s actionable.
+- If the title doesn’t communicate the issue, edit it for clarity.
+- If you feel comfortable triaging the issue further, then you can also:
+  - Check that the bug report is valid by debugging it to see if you can track down the technical specifics.
+  - Check if the issue is missing some detail and see if you can fill in those details. For instance, if a bug report is missing visual detail, it’s helpful to reproduce the issue locally and upload a GIF.
+
+
+## Pull Requests
+
+Gutenberg follows a feature branch pull request workflow for all code and documentation changes. At a high-level, the process looks like this:
+
+
+1. Check out a new feature branch locally.
+2. Make your changes, testing thoroughly.
+3. Commit your changes when you’re happy with them, and push the branch.
+4. Open your pull request.
+
+Along with this process, there are a few important points to mention:
+
+- Non-trivial pull requests should be preceded by a related issue that defines the problem to solve and allows for discussion of the most appropriate solution before actually writing code.
+- To make it far easier to merge your code, each pull request should only contain one conceptual change. Keeping contributions atomic keeps the pull request discussion focused on one topic and makes it possible to approve changes on a case-by-case basis.
+- Separate pull requests can address different items or todos from their linked issue, there’s no need for a single pull request to cover a single issue if the issue is non-trivial.
+
+**Code Review**
+
+Every pull request goes through a manual code review, in addition to automated tests. The objectives for the code review are best thought of as:
+
+- Correct — Does the change do what it’s supposed to?
+- Secure — Would a nefarious party find some way to exploit this change?
+- Readable — Will your future self be able to understand this change months down the road?
+- Elegant — Does the change fit aesthetically within the overall style and architecture?
+- Altruistic — How does this change contribute to the greater whole?
+
+*As a reviewer*, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.
+
+*As a contributor*, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
+
```

Comment:
Maybe we could add some bits about:

- Anyone can do a code review, even newcomers.
- GitHub's code review workflows ("Request Changes" vs. "Approve" vs. "Comment")

Reply:
+1 for showing GitHub's review workflow here and also using as a call for encouraging new reviewers.